packages/contracts-bedrock/README.md

```diff
@@ -138,3 +138,19 @@ Unless explicitly discussed otherwise, you MUST include the following basic upgr
 2. Include a `uint8 public constant VERSION = X` at the TOP of your contract.
 3. Include a function `initialize` with the modifier `reinitializer(VERSION)`.
 4. In the `constructor`, set any `immutable` variables and call the `initialize` function for setting mutables.
+
```

Comment:
I have a slight preference for adding this to the specs over the readme

Reply:
Can do that later, I think it's fine here for now